cmake/configure/configure_20_boost.cmake

```diff
@@ -42,7 +42,7 @@ macro(feature_boost_find_external var)
       # Test that Boost.Iostreams is usable.
       #
       reset_cmake_required()
-      list(APPEND CMAKE_REQUIRED_LIBRARIES ${BOOST_LIBRARIES})
+      list(APPEND CMAKE_REQUIRED_LIBRARIES ${BOOST_TARGETS})
```

Comment:
This is the main problem: Supporting import targets in `check_cxx_source_compiles` has been fixed at some point... But I am not sure when.